### PR TITLE
[Doc Improvement][Update TeamsFx CLI Doc][2409882]

### PR DESCRIPTION
**Pull request by MuyangAmigo**
Updates to [TeamsFx CLI Doc](https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/teamsfx-cli) to prepare for TeamsFx Build update.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c4920e534dd63b723cf18bd418421bd4f769ca7d/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 86 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6e9902a0-0b9f-4197-9977-a6e20bea1073) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/66d8026484cf8cf39d38d9a468189003691989f5/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 85 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cd0d70a1-999c-4430-b759-d5b9725b27ff) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/37c335bf7d362072c69798e8400a8ee8c8a6c534/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 85 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fb1154bc-c5b0-4c93-a941-6481325800aa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6f3f120e5754a5d7e9691d964b4fd86926cc0a0c/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 85 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/07e29a57-8f8f-4a66-abd8-8ce094d723ba) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b2763a5a18dfd645a3284faa1f7de0c089b76dec/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 84 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8de52cc5-51c3-4064-a367-8faa990c6387) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e3e139f3f74371de9042aaefe63007963c79b4f/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 84 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ded630d7-3da6-4c77-9c9d-0c1d909a7697) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c243b2dbb28d795e775807b81ecadd94f4964673/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 84 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3201bc8a-dba4-41b9-bdc7-43088db627fb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/TeamsFx-CLI.md [hunk 1]
```diff
@@ -71,126 +69,90 @@ By default, `teamsfx new` goes into interactive mode and guides you through the
...
 ## `teamsfx account`
 
-Manage cloud service accounts. The supported cloud services are `Azure` and `Microsoft 365`.
+The following table lists the cloud service accounts,such as Azure and Microsoft 365.
 
-| `teamsFx account` Command | Description |
+| Command | Description |
 |:----------------  |:-------------|
 | `teamsfx account login <service>`  | Log in to the selected cloud service. |
 | `teamsfx account logout <service>`  | log out of selected cloud service. |
```
Comment (zyxiaoyuer) [inline msteams-platform/toolkit/TeamsFx-CLI.md:128]: | `teamsfx account login <service>`  | Log in to the selected cloud service. Service options are M365 or Azure. |
| `teamsfx account logout <service>`  | log out of selected cloud service. Service options are M365 or Azure. |

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/47a484f1e8413f366a8594d01e9111f8601f204e/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 83 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f91aed2e-5f3a-42db-9814-818c2fab27b7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/34d98317c39baa9a52df32f18daa1dcb2caca431/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 81 | 40 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/25a237e1-4e00-4b24-978c-a8e6487cae4d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/TeamsFx-CLI.md (hunk 1 above)
Comment (v-abirade) [reply msteams-platform/toolkit/TeamsFx-CLI.md]: Updated as suggested.

**Review by zyxiaoyuer — APPROVED** [review]

**Pull request merged**